docs/install/create-project.md

```diff
@@ -134,15 +134,14 @@ For the `fish` shell, see [`nvm.fish`](https://github.com/jorgebucaran/nvm.fish)
 
 (install-prerequisites-yeoman-label)=
 
-#### Yeoman
+#### Yeoman and the Volto boilerplate generator
 
 Install {term}`Yeoman`.
 
 ```shell
-npm install -g yo
+npm install -g yo @plone/generator-volto
 ```
 
-
 (install-prerequisites-yarn-label)=
 
 #### Yarn 3
```

Comment:
Yarn 3 or 4 or just Yarn?

Reply:
Also check line 149. I think we can drop the parenthetical statement by now.

Reply:
`````suggestion
#### Yarn

````important
These instructions will not work if you have used another package manager, such as Homebrew on macOS, to install Yarn.
You can verify where you installed Yarn and its version.

```shell
which yarn
```
```console
/opt/homebrew/bin/yarn
```

```shell
yarn -v
```
```console
3.2.3
```

If the console includes `homebrew` in the path, and the version of Yarn is not supported, then you must uninstall it.

```shell
brew uninstall yarn
```

Now the instructions should work.
````
`````